### DISCUSSION TITLE
参考文献编号的空隙如何调整

### DISCUSSION THREAD

**Discussion by sqinghua** [sjtug/SJTUThesis#1029] [提问] [answered]
参考文献100多条，目前编译出来的编号中，方括号和数字之间的间隔很大，看样子是为了与100多编号的占位对齐，如下图：
![屏幕截图 2024-04-20 103103](https://github.com/sjtug/SJTUThesis/assets/71050713/0d95a124-2f4d-4e69-91d8-0f6da6e0510b)
如何调整为下图所示：
![屏幕截图 2024-04-20 103129](https://github.com/sjtug/SJTUThesis/assets/71050713/7ed9aec0-7bdb-4d52-a8b5-0155633e9537)

非常感谢！

**Comment by AlexaraWu** [comment, marked as answer]
你 `biblatex-gb7714-2015` 是不是设置了 `gbalign=center` 选项？默认是 `gbalign=right`。

**Reply by sqinghua** [reply]
感谢！！
